CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - zone label support for Tarantool Cartridge >= '2.4.0'
 
+### Fixed
+- `ipairs` instead of `pairs` while iteration in `histogram`
```

Comment:
Why is this a fix?

Reply:
1) `pairs` not guarantee iteration order 
2) `pairs` can't be optimized by JIT